miner/algo_greedy.go

```diff
@@ -31,10 +34,12 @@ func newGreedyBuilder(chain *core.BlockChain, chainConfig *params.ChainConfig, b
 	}
 }
 
-func (b *greedyBuilder) mergeOrdersIntoEnvDiff(envDiff *environmentDiff, orders *types.TransactionsByPriceAndNonce) ([]types.SimulatedBundle, []types.UsedSBundle) {
-	usedBundles := []types.SimulatedBundle{}
-	usedSbundles := []types.UsedSBundle{}
-
+func (b *greedyBuilder) mergeOrdersIntoEnvDiff(
```

Comment:
I find this formatting confusing, it's not present elsewhere in the codebase - I'd either stick to the codebase's style, or make the formatting a bit more obvious
```
func (b *greedyBuilder) mergeOrdersIntoEnvDiff(
	envDiff *environmentDiff, orders *types.TransactionsByPriceAndNonce) ([]types.SimulatedBundle, []types.UsedSBundle
) {
```

Reply:
fair, updated